### PR TITLE
784: Add first draft front end files and tests

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

File: app/views/caseload.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+{% include "includes/validation-error-messages.html" %}
+
+{% include "includes/title-subtitle.html" %}
+
+{% include "includes/sub_nav.html" %}
+
+<h3 class="heading-medium">Overall caseload by tier</h3>
+
+<table class="sln-table-caseload data-table">
```
Comment (dconey646) [inline app/views/caseload.html]: Could we extra this to a caseload table includes? Just to follow suit from the rest of the main pages.

File: test/accessibility/run-pa11y.sh [hunk 2]
```diff
@@ -51,6 +51,10 @@ urls+=("$host/ldu/$lduId/$overview_url")
 urls+=("$host/region/$regionId/$overview_url")
 urls+=("$host/hmpps/0/$overview_url")
 
+# Caseload
+caseload_url='caseload'
+urls+=("$host/team/$teamId/$caseload_url")
+
```
Comment (dconey646) [inline test/accessibility/run-pa11y.sh]: 👍

File: app/views/caseload.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  <thead>
+    <tr>
+      <th colspan="2">Offender manager</th>
+      <th colspan="9">Case tiers</th>
```
Comment (tsheppard01) [inline app/views/caseload.html]: Pa11y is gonna cry about this, need to have an id and a header inside all your <th> tags.  You can look at the organisation-overview.html file to see how its done

File: test/e2e/view-caseload.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  it('should navigate to the team caseload screen', () => {
+    return browser.url('/team/' + workloadOwnerIds.filter((item) => item.table === 'team')[0].id + '/caseload')
+      .waitForExist('.breadcrumbs')
+      .waitForExist('.sln-subnav')
+      .waitForExist('[href="/team/' + workloadOwnerIds.filter((item) => item.table === 'team')[0].id + '/overview"]')
+      .waitForExist('[href="/team/' + workloadOwnerIds.filter((item) => item.table === 'team')[0].id + '/caseload-capacity"]')
+      .waitForExist('[href="/team/' + workloadOwnerIds.filter((item) => item.table === 'team')[0].id + '/case-progress"]')
+      .waitForExist('.sln-table-caseload')
+      .waitForExist('.sln-page-subtitle')
```
Comment (tsheppard01) [inline test/e2e/view-caseload.js]: Can you add a waitForExist for the offender manager url?

File: app/views/caseload.html (hunk 1 above)
Comment (caoimheb) [inline app/views/caseload.html]: Was wondering about this. Only overview is separated out into includes, case progress and capacity are embedded.

File: app/views/caseload.html (hunk 3 above)
Comment (caoimheb) [inline app/views/caseload.html]: Updated

File: test/e2e/view-caseload.js (hunk 4 above)
Comment (caoimheb) [inline test/e2e/view-caseload.js]: Added

**Comment by tsheppard01** [comment]
LGTM 👍

**Pull request merged**